src/application/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ endif()
 find_package(Boost 1.53.0 REQUIRED COMPONENTS program_options system filesystem)
```

Comment:
@simogasp Do we need find_package(Boost ...) twice for this fix to work?

Reply:
yes because the application can be an independent cmake project (you can use popsift installed somewhere else to build, so you can test the use as 3rd pary) and requires a different set of components from boost.

Reply:
oh no, sorry i didn't see that, i thought you were referring to the one in the main cmakefile. Obviously this is unnecessary, i'll fix it